client/client.go

```diff
@@ -67,6 +68,28 @@ type TargetConf struct {
 	macros map[string]string
 }
 
+type TerminalConf struct {
+	Color bool
+}
+
+//Used for color print in terminal base on color flag
+//in configuration file
+func (clr TerminalConf) Printf(c string, s string, a ...interface{}) {
+	if clr.Color {
+		fmt.Printf("\x1b["+c+"m"+s+"\x1b[0m", a...)
```

Comment:
off hand looking at this, does this assume the entire line will utilize the same terminal color/attributes? does this cause problems in place where we have various highlighting currently on the same line.

given the current code is it possible for us to just wrap the output string in a function that strips the control codes if colorized output has been disabled by the user? That would cover off all the requires cases